website/content/api-docs/system/secrets-sync.mdx

```diff
@@ -304,6 +338,15 @@ This endpoint creates a destination to synchronize secrets with an Azure Key Vau
 
 - `cloud` `(string: "cloud")` - Specifies a cloud for the client. The default is Azure Public Cloud.
 
+- `secret_name_template` `(string: "")` - Template to use when generating the secret names on the external system.
+The default template yields names like `vault-kv-1234-my-secret`. See [this documentation](/vault/docs/sync#name-template) for more details.
+
+- `custom_tags` `(map<string|string>: nil)` - List of custom tags or labels to set on the synced secrets at the destination.
+Custom tags are merged with internal built-in tags.
```

Comment:
Do we call out the internal tags anywhere in the docs?

Reply:
I don't think so yet. It'll be mentioned in the section about how to make the secrets read-only to prevent out-of-band overwrites. The built-in tags are the condition to use so Vault can only access these secrets or other principals cannot access secrets managed by Vault.